cmd/mqtt/main.go

```diff
@@ -14,14 +14,16 @@ import (
 
 	"github.com/go-redis/redis"
 	"github.com/mainflux/mainflux"
-	"github.com/mainflux/mainflux/broker"
 	"github.com/mainflux/mainflux/logger"
+	"github.com/mainflux/mainflux/messaging"
+	pubsub "github.com/mainflux/mainflux/messaging/nats"
```

Comment:
same here

Reply:
I renamed to `messaging`, but here I need both `messaging` and `messaging/nats`. Same for `Twins` service.

Reply:
Why not then calling it `nats.NewPublisher` - it will be clearer from the code that this is a specific NATS-based implementation of our interface.

Reply:
TBH, actually the best would be probably something like `messagingNats.NewPublisher`, to distinguish from ordinary NATS package and not to be confused with functions from NATS driver.

I can accept `pusub` alias as well, just a proposition.

Reply:
I removed named import by addressing this one: https://github.com/mainflux/mainflux/pull/1141#discussion_r415905298.